### PR TITLE
Remove ramda.

### PR DESCRIPTION
**Pull request by wtgtybhertgeghgtwtg**
Removes ramda as a dependency.

### CODE REVIEW

File: src/createModule/index.js [hunk 1]
```diff
@@ -1,28 +1,21 @@
...
 
-const _generateReduxComponents = (name, initialState) => transformations => {
-  const generated = {
+// eslint-disable-next-line new-cap
+export const createModule = ({ name, transformations, initialState = Map() }) => {
+  const formatTransformation = transformation => ({
+    ...transformation,
+    formattedConstant: `${name}/${transformation.action}`,
+  });
+  const formattedTransformations = transformations.map(formatTransformation);
```
Comment (mboperator) [inline src/createModule/index.js]: This is much cleaner

File: src/createModule/payloadPropchecker.js [hunk 2]
```diff
@@ -10,20 +8,23 @@ const defaultOnError = err => {
...
-      const propChecker = payloadTypes[type] || defaultPropCheck;
-      const typeError = propChecker(payload, type, formattedConstant, 'prop') || {};
-      const { message } = typeError;
-
-      message && onError(message);
+    if (typeof payloadTypes === 'undefined') {
+      return { payload, meta };
+    }
+    const keys = Object.keys(payloadTypes);
+    for (let i = 0; i < keys.length; ++i) {
```
Comment (mboperator) [inline src/createModule/payloadPropchecker.js]: I'd prefer that this remained `forEach` for consistency's sake

Comment (wtgtybhertgeghgtwtg) [inline src/createModule/payloadPropchecker.js]: `Array.forEach` has consistently and measurably worse performance than native `for` loops.  I can if you want, but it'd be less performant.

Comment (mboperator) [inline src/createModule/payloadPropchecker.js]: Good to know!

**Comment by mboperator** [comment]
+1

**Comment by mboperator** [comment]
Do you mind merging master into this?

**Comment by wtgtybhertgeghgtwtg** [comment]
master merged.

**Pull request merged**